.gitignore

```diff
@@ -92,7 +92,7 @@ venv.bak/
 .idea/
 
 # VSCode project settings
-.vscode/
+/.vscode
```

Comment:
Is this only ignoring the top-level `.vscode` directory now?

Reply:
Yes. There is the `.gitpod/.vscode` which I want to track with git (that's why putting the `/` to the front). Then, Gitpod will setup a symlink from `.vscode` to `.gitpod/.vscode`, therefore I have to remove the trailing `/` as otherwise the symlink is not ignored.